facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java

```diff
@@ -275,7 +275,13 @@ private boolean shouldCheckConsistencyConsistency(FacadeOperation facadeOperatio
             try {
                 legacyEntity = (T) method.invoke(legacySvc, values);
             } catch (InvocationTargetException e) {
-                throw e.getCause();
+                if (shouldDestination(facadeOperation) && !shouldCheckConsistency(facadeOperation)) {
+                    // Lightblue is going to be called and consistency checker is disabled
+                    // swallow the exception from legacy
+                    log.warn("Legacy call " + implementationName + "." + callStringifier + " threw an exception. Returning data from Lightblue.", e.getCause());
+                } else {
+                    throw e.getCause();
```

Comment:
Should we cancel the lightblue call future if it is in progress at this point? That being said I realize this is same as previous behavior.